bluesky_queueserver/manager/start_manager.py

```diff
@@ -260,38 +274,54 @@ def start_manager():
         config_worker["kafka"]["topic"] = args.kafka_topic
         config_worker["kafka"]["bootstrap"] = args.kafka_server
 
-    if args.profile_collection_path:
-        pc_path = args.profile_collection_path
-        pc_path = os.path.abspath(os.path.expanduser(pc_path))
-        if not os.path.exists(pc_path):
-            logger.error("Profile collection directory '%s' does not exist", pc_path)
-            return 1
-        if not os.path.isdir(pc_path):
-            logger.error("Path to profile collection '%s' is not a directory", pc_path)
+    # Find startup directory
+    if args.profile_name:
+        profile_name = args.profile_name
+        try:
+            import IPython
+
+            path_to_ipython = IPython.paths.get_ipython_dir()
+        except Exception:
+            logger.error(
+                "IPython is not installed. Specify directory to startup file by using " "'--startup-dir' option."
+            )
             return 1
+        startup_dir = os.path.abspath(path_to_ipython)
```

Comment:
This is redefined below to mean something else. Redefining local variables within the same scope and branch can make debugging confusing. Can we rename this variable `ipython_dir`?

Reply:
Done.